source/_components/huawei_lte.markdown

```diff
@@ -122,8 +122,8 @@ sensor:
     monitored_conditions:
```

Comment:
Monitored conditions should be removed overall, for more information see ADR-0003:

https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

Reply:
Sure, I know, but these are just some changes I need to get out of the way before making larger changes. There will be quite a few, unique ids will change, all config will consolidate below top level huawei_lte entry, etc.

I will look into monitored_conditions as part of that, but I must also note that the ADR says "if that data is fetched in a single API request". In huawei_lte's case, it is not -- each category of information requires a separate http request to the router, and even worse, the set of things available from the same requests varies to some extent between routers. But as said, that's something for another near future changeset.